modules/nw-dns-forward.adoc

```diff
@@ -5,7 +5,16 @@
 [id="nw-dns-forward_{context}"]
 = Using DNS forwarding
 
-You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+You can use DNS forwarding to override the default forwarding configuration in the `/etc/resolv.conf` file in the following ways:
+
+* Specifying name servers for every zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+* Providing a list of upstream DNS servers.
+* Changing the default forwarding policy.
```

Comment:
I'd consider changing these to imperatives. Specify, Provide, Change.

Reply:
@ctauchen comment addressed.